plugin-examples/src/main/java/com/hotels/styx/ExamplePlugin.java

```diff
@@ -1,5 +1,5 @@
 /*
-  Copyright (C) 2013-2018 Expedia Inc.
+  Copyright (C) ${license.git.copyrightYears} Expedia Inc.
```

Comment:
Where did this come from? Appears in 3 more classes below too.

Reply:
There is something in `maven` that keeps automatically changing these 4 files every time I'll do a build from the command line. I haven't had time to investigate this issue. But obviously I should have backed off the unintended changes before publishing a PR.

Well these unintended changes have now been backed off.